src/actions/websocket.js

```diff
@@ -355,19 +366,20 @@ function handleUpdateTeamEvent(msg) {
     };
 }
 
-function handlePatchTeamEvent(msg) {
-    return {
-        type: TeamTypes.PATCHED_TEAM,
-        data: JSON.parse(msg.data.team),
-    };
-}
+// function handlePatchTeamEvent(msg) {
+//     return {
+//         type: TeamTypes.PATCHED_TEAM,
+//         data: JSON.parse(msg.data.team),
+//     };
+// }
```

Comment:
There is no WebsocketEvents.PATH_TEAM constant present. I commented out the handle function. But it is rather needs to be removed. Comments welcome

Reply:
Understood. Please remove it.

Reply:
All done. The snipped is `Outdated`. All commented out code is already deleted.